core/Tracker/Visit/VisitProperties.php

```diff
@@ -42,7 +53,7 @@ public function getProperty($name)
      *
      * @return array
      */
-    public function &getProperties()
+    public function &getProperties(): ?array
```

Comment:
How could that one become null?
```suggestion
    public function &getProperties(): array
```